.travis.yml

```diff
@@ -9,6 +9,9 @@ env:
   - DJANGO='django<1.5'
   - DJANGO='django<1.6'
   - DJANGO='django<1.7'
+  - DJANGO='django<1.8'
+  - DJANGO='django<1.9'
```

Comment:
This will to intall 1.9b1 but 1.8. In its default configuration tox ignores development versions. Pin the exact version on PyPi to installit (1.9b1)
Ditto for 1.10